src/openagi/llms/groq.py

```diff
@@ -26,10 +21,8 @@ class GroqModel(LLMBaseModel):
 
     def load(self):
         """Initializes the GroqModel instance with configurations."""
-        self.llm = ChatGroq(
-            model_name = self.config.model_name,
-            groq_api_key = self.config.groq_api_key,
-            temperature = self.config.temperature
+        self.llm = Groq(
```

Comment:
Try to keep this code within try-except. If API key is not detected, an exception needs to be triggered